src/main/java/org/jabref/logic/cleanup/FieldFormatterCleanup.java

```diff
@@ -105,15 +105,15 @@ public Formatter getFormatter() {
     }
 
     @Override
-    public boolean equals(Object o) {
-        if (this == o) {
+    public boolean equals(Object obj) {
+        if (this == obj) {
             return true;
         }
-        if (o instanceof FieldFormatterCleanup) {
-            FieldFormatterCleanup that = (FieldFormatterCleanup) o;
-            return Objects.equals(field, that.field) && Objects.equals(formatter, that.formatter);
+        if (!(obj instanceof FieldFormatterCleanup)) {
+            return false;
         }
-        return false;
+        FieldFormatterCleanup other = (FieldFormatterCleanup) obj;
+        return Objects.equals(field, other.field) && Objects.equals(formatter, other.formatter);
     }
```

Comment:
The new code seems to be the equial to the old one. ^^.

Reply:
I just used eclipse to regenerate it to have it consistent with others